docs/dev/setup/local-database-tests.rst

```diff
@@ -0,0 +1,37 @@
+.. _local_database_tests:
+
+Running database tests locally
+---------------------------------
+
+Prerequisites
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Database tests are standard server integration tests executed against real database (MySQL or Postgres) started as Docker containers. Gradle test task will automatically pull necessary images and start Docker containers.
+
+Before running those tests make sure that the following steps were executed:
```

Comment:
Missing "," after "those tests".
Please consider not using passive voice and therefore rephrasing the "Before running..." sentence.